Documentation-Review-Process.md

```diff
@@ -0,0 +1,164 @@
+* [Lightweight document review process](#lightweight-document-review-process)
+
+## Introduction
+
+The process for reviewing documentation PRs is slightly different from
+[the general process for reviewing code PRs](https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#reviewsPR-Review-Guide.md).
+This document explains the differences.
+
+## The Documentation Team
+
+The Documentation Team are responsible for carrying out a full documentation
+review of a PRs containing documentation changes. Such a review includes
+checking the grammar, structure, consistency, and typographical correctness of
+the documentation changes.
+
+The following GitHub team lists the user users who are members of the Documentation Team:
+
+- https://github.com/orgs/kata-containers/teams/documentation/members
+
+## Default document review process
+
+[Project maintainers](https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#project-maintainers)
+have the power to approve or reject PRs using the pullapprove tool employed by
+all repositories.
+
+If a Documentation Team review is required, the pullapprove configuration will
+apply and the PR will be blocked until the required number of acks are
+obtained for the documentation changes.
+
+The technical writer from the Documentation Team will then review the document
+changes and add comments explaining how wording should be changed. It is then
+up to the PR creator to apply these changes to their original PR, rebasing as
+necessary to ensure a clean `git(1)` history. Once the creator has applied the
+changes they need to re-push their branch and add a comment to the PR asking
+for a re-review. This process can continue effectively indefinitely until the
+Documentation Team is happy with the final result.
+
+## Justification for different process
+
+The documentation review process is different for the code review process for
+the following reasons:
+
+### Users should always expect accurate Documentation
+
+Just like code changes, some documentation changes have to be merged as soon
+as possible to ensure users have the most current and accurate information.
+
+But documentation changes tend to be more urgent - consider a "Priority 1"
+code issue. This will need to be fixed with all urgency. But it would also
+need to be documented clearly (and quickly). If the P1 issue cannot be fixed
+quickly, the documentation needs to be changed fast to ensure users are aware
+of the issue and are provided with suitable workarounds.
+
+Turning the problem round, if documentation is *not* kept current, users will
+have incorrect information and may end up creating problems for themselves, or
+worse become disaffected with the project.
+
+In summary, documentation PRs should not sit in the PR backlog for any longer
+than necessary.
+
+### Some documentation PRs can be fast-tracked
+
+If a PR makes a minor change to one or more documents, it may not justify
```

Comment:
If a PR makes a minor change to one or more documents, passing to the Documentation Team for a formal review and approval might not be necessary.